pom.xml

```diff
@@ -29,9 +29,16 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-platform-bom</artifactId>
+        <version>${version.org.kie}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
         <groupId>org.drools</groupId>
         <artifactId>drools-bom</artifactId>
-        <version>${drools.bom.version}</version>
+        <version>${version.org.kie}</version>
         <type>pom</type>
```

Comment:
@baldimir I have question about this - should we use jbpm + drools BOM or whole kie-platform-bom?

Reply:
We can leave just kie-platform-bom there. It will be better, because also OptaPlanner tests would be migrated sometime in the future eventually.